e2e/LongHaul/device/IotHub.cs

```diff
@@ -128,6 +129,31 @@ public async Task RunAsync(CancellationToken ct)
             }
         }
 
+        public async Task CheckDesiredPropertiesAndUpdateAsync(CancellationToken ct)
```

Comment:
This should just be about setting a read-only property in a loop. It can be based on something, like how many telemetry messages have been sent or something, or it could be gibberish, like a Guid.